modules/gathering-operator-logs.adoc

```diff
@@ -12,7 +12,7 @@ If you experience Operator issues, you can gather detailed diagnostic informatio
 * You have access to the cluster as a user with the `cluster-admin` role.
 * Your API service is still functional.
 * You have installed the OpenShift CLI (`oc`).
-* You have the fully qualified domain names of the control plane, or control plane machines (also known as the master machines).
+* You have the fully qualified domain names of the control plane, or control plane machines.
```

Comment:
I think this sentence needs to be fixed too, to just say "names of the control plan machines".

(This was technically an issue even before your update)

Reply:
Thanks. Wasn't sure that was right!